telluric/georaster.py

```diff
@@ -1561,8 +1561,9 @@ def get_tile(self, x_tile, y_tile, zoom,
         bands = bands or list(range(1, self.num_bands + 1))
         # we know the affine the result should produce becuase we know where
         # it is located by the xyz, therefore we calculate it here
-        resolution = MERCATOR_RESOLUTION_MAPPING[zoom]
-        affine = Affine.translation(coordinates.left, coordinates.top) * Affine.scale(resolution, -resolution)
+        ratio = MERCATOR_RESOLUTION_MAPPING[zoom]/self.resolution()
```

Comment:
that is still wrong because you losing direction of the scaling, I would do:
a_ratio = MERCATOR_RESOLUTION_MAPPING[zoom]/self.affine[0]
e_ratio = MERCATOR_RESOLUTION_MAPPING[zoom]/self.affine[4]

affine = self.window_transform(window)
affine = affine * Affine.scale(a_ratio, e_ratio)